.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,24 @@
+name: Unit tests
```

Comment:
whats the difference between this one and the .github/workflows/tests.yaml?

Reply:
This one is the one coming from the `ros-installer`. I realized both tests are not colliding and the former tests.yaml from `elemental-operator` is broken. Hence I did not want to mess with that in this PR, simply add the new logic without altering whatever it was already there.

Reply:
sounds good, I will merge them together on my PR :+1: